src/Microsoft.Health.CosmosDb/Features/Storage/CosmosDbSubStatusValues.cs

```diff
@@ -7,6 +7,16 @@ namespace Microsoft.Health.CosmosDb.Features.Storage
 {
     public static class CosmosDbSubStatusValues
```

Comment:
`KnownCosmosDbSubStatusValues` would follow the convention we've used in other parts of the codebase for these types of specified values